thirdparty/mupdf/external_fonts.patch

```diff
@@ -68,7 +68,7 @@ index f6951ba2..b1fdd981 100644
  /*
  	Base 14 PDF fonts from URW.
  	Noto fonts from Google.
-@@ -367,3 +371,185 @@ fz_lookup_noto_emoji_font(fz_context *ctx, int *size)
+@@ -367,3 +371,188 @@ fz_lookup_noto_emoji_font(fz_context *ctx, int *size)
```

Comment:
@NiLuJe Apparently I thought your suggestion took off more lines than it did? :-P Oh well, luckily there's the CI.